website/docs/registry/providers/tiers.html.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: "registry"
+page_title: "Terraform Registry - Provider Tiers
+sidebar_current: "docs-registry-provider-tiers
+description: |-
+  Published Provider tiers in the Terraform Registry
+---
+
+# Provider Tiers
+
+There are three tiers of providers in the Terraform Registry:
+
+* **Official Providers** - are built, signed, and supported by HashiCorp. Official Providers can typically used without providing
+  provider source information in your Terraform configuration.
+* **Partner Providers** - are built, signed, and supported by a third party. HashiCorp has verified the ownership of the private
+  key and we provide a chain of trust to the CLI to verify this programatically. To use Partner Providers in your Terraform
+  configuration, you need to specify the namespace and name to download from the registry.
```

Comment:
It's more accurate (but not necessarily clearer) to say that provider source is required for partner providers. Only the namespace and provider name are required, but that's specifically in the `"source"` attribute under `terraform.required_providers`.  (this isn't really a suggestion and I won't think twice if you don't do anything with this information)

Reply:
```suggestion
  configuration, you need to specify the provider source, typically this is the namespace and name to download from the registry.
```